### DISCUSSION TITLE
How is this nav bar implemented on shadui landing page, couldn't find the relevant components

### DISCUSSION THREAD

**Discussion by Pushan2005** [shadcn-ui/ui#2913] [Q&A]
![image](https://github.com/shadcn-ui/ui/assets/95559976/54c6e87a-0e8a-42c4-9631-ea14e3527655)

**Comment by pablotanner** [comment]
You can get this quite easily by using the Tab Component and restyling it a bit.

**Reply by Pushan2005** [reply]
thank you
